package.json

```diff
@@ -26,7 +26,7 @@
     "@coingecko/cryptoformat": "^0.5.4",
     "@loadable/component": "^5.15.2",
     "@oceanprotocol/art": "^3.2.0",
-    "@oceanprotocol/lib": "^2.2.3",
+    "@oceanprotocol/lib": "^2.3.0",
```

Comment:
need to run `npm install` so `package-lock.json` updates, then commit that file too for actual upgrade